keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -690,7 +696,13 @@ and restarting the component. No other changes should be necessary to
 disable the feature.
 
 NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
---> Yes, the feature may be disabled by reverting the feature flag.
+-->
+
+Yes, the feature may be disabled on the apiserver by reverting the
```

Comment:
Can you add a sentence explaining that this means that if there is a client bug, either the application needs to be re-compiled or the server needs to disable the feature until the clients are fixed.

I'm ok with this, but we want to be sure cluster-admins can understand it.

Reply:
Updated